interactive/api/static/examples/FULL_GAS_PHASE/camp_data/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "camp-files" : [
-    "camp_data/tolerance.json",
     "camp_data/species.json",
-    "camp_data/reactions.json"
+    "camp_data/reactions.json",
+	"camp_data/tolerance.json"
```

Comment:
A minor thing, but can we use the uniform indentation?

Reply:
I'll try tabs in Notepad++.